packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -2137,13 +2137,29 @@
   "Keep_default_user_settings": "Keep the default settings",
   "Keyboard_Shortcuts_Edit_Previous_Message": "Edit previous message",
   "Keyboard_Shortcuts_Keys_1": "Command (or Ctrl) + p OR Command (or Ctrl) + k",
+  "Keyboard_Shortcuts_Keys_1_win": "Ctrl + p OR Ctrl + k",
+  "Keyboard_Shortcuts_Keys_1_mac": "Command + p OR Command + k",
```

Comment:
Is it posible to use the ⌘ symbol rather than `Command`. According to [apple's documentation](https://developer.apple.com/design/human-interface-guidelines/macos/user-interaction/keyboard/) the symbol should be used rather than the name of the key. Also symbols are universal. Same with arrows, shift. and option

Reply:
Yeah, you are right. I too see it would be better to the `⌘` symbol.
But, you see I have just changed the added what the previous *key-value* of **Keyboard Shortcuts** where indicating. :sweat_smile: . 
If the collaborators say so, I would do that.